src/main/resources/test_files/runtime-spark/RumbleML/MLTransformerMultiple1.jq.disabled

```diff
@@ -1,8 +1,8 @@
 (:JIQS: ShouldRun; Output="" :)
 let $tokenizer := get-transformer("Tokenizer")
 let $hashingTF := get-transformer("HashingTF")
-let $data := (structured-json-file("./src/main/resources/queries/rumbleML/sample-ml-numeric-data.json"))
-let $intermediate := $tokenizer($data, {"inputCol": "values", "outputCol": "output"})
+let $data := structured-json-file("./src/main/resources/queries/rumbleML/sample-ml-string-data.json")
+let $intermediate := $tokenizer($data, {"inputCol": "sentence", "outputCol": "output"})
 return $hashingTF($intermediate, {"inputCol": "output", "numFeatures": 2})
```

Comment:
This test is still disabled. 
Result of 1st transformer contains a column of Vector type. This vector type is converted to an ObjectItem and stored. However, for second operation this ObjectItem needs to be converted back into a native Java type so that it can be processed by SparkML. This is not implemented yet at the time of this comment.